rust/core-lib/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "xi-core-lib"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
Ah, just one more thing: Do you plan to do additional changes? If so, it might be worth it to delay 0.4 until after you made those.

Reply:
No, I'm not planning any other changes at the moment.